BitwardenShared/Core/Platform/Services/EventServiceTests.swift

```diff
@@ -6,7 +6,9 @@ class EventServiceTests: XCTestCase {
     // MARK: Properties
 
     var cipherService: MockCipherService!
+    var client: MockHTTPClient!
     var errorReporter: MockErrorReporter!
+    var eventAPIService: APIService!
```

Comment:
⛏️ Could you `nil` these out in `tearDown()`?

Reply:
Good catch!